scripts/add_user_to_group.py

```diff
@@ -0,0 +1,81 @@
+import argparse
```

Comment:
If this is being run as a script it will be worth adding a shebang

Reply:
agreed